### PR TITLE
feat: cover base level packages #1967

### LINKED ISSUE #1967: Expand changeScanner to cover base level packages
**Is your feature request related to a problem? Please describe.**
Currently `npm run changeScanner` will complain if you make a change to a package without updating its changelog/package/packageLock.

**Describe the solution you'd like**
Update changeScanner so if someone makes a change to psammead, outside of the `packages` folder it complains you havent bumped the base level psammead changelog/package/packageLock

E.g. changing `psammead/scripts/enforceVersions.sh` would complain and ask you top bump the base level changelog/package/packageLock

but

changing `psammead/packages/utilities/gel-foundations/breakpoints/package.json` would complain that you havent updating the gel-foundations changelog/package/packageLock, NOT the base level ones

**Testing notes**
Write unit tests

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by mykeels**
Resolves #1967

**Overall change:** Expand changeScanner to cover base level packages.

**Code changes:**

- _Use `git diff --name-only origin/latest` alone instead of with `/packages/*/*/*` suffix._
- _Update tests to adjust for the change._
- _Update psammead version to `1.7.14`._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM, slight bug however, editing `psammead/packages/components/README.md` says
```
Branch must update CHANGELOG.md in README.md
Branch must update package-lock.json in README.md
Branch must update package.json in README.md
```
When it should be covered by `psammead`

**Review by dr3 — APPROVED** [review]
LGTM, great work

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
-| 1.7.11 | [PR#2090](https://github.com/bbc/psammead/pull/2090) Talos - Bump Dependencies - @bbc/psammead-locales |
+| 1.7.13 | [PR#2090](https://github.com/bbc/psammead/pull/2090) Talos - Bump Dependencies - @bbc/psammead-locales |
```
Comment (dr3) [inline CHANGELOG.md:14]: Not sure whats happend to this change log, looks like your going from `1.7.11` -> `1.7.13`?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "1.7.11",
+  "version": "1.7.13",
```
Comment (dr3) [inline package.json]:
```suggestion
  "version": "1.7.12",
```

**Review by dr3 — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

File: scripts/changeScanner/getChanges.test.js [hunk 3]
```diff
@@ -13,6 +13,13 @@ describe(`changeScanner - getChanges`, () => {
         'packages/components/foobar/index.test.js',
         'packages/components/barfoo/package.json',
         'packages/components/apples/dist/package.json',
+        'packages/containers/foo-bar/dist/package.json',
+        'packages/utilities/bar-foo/dist/package.json',
+        'scripts/getChanges.js',
+        'scripts/changeScanner/index.js',
+        'psammead/packages/components/README.md',
+        'packages/components/README.md',
```
Comment (thekp) [inline scripts/changeScanner/getChanges.test.js:20]: is `'packages/components/README.md'` , not the same as `'psammead/packages/components/README.md',` 

??

Comment (mykeels) [reply scripts/changeScanner/getChanges.test.js:20]: Fixed!

Comment (thekp) [reply scripts/changeScanner/getChanges.test.js:20]: thanks

**Review by thekp — APPROVED** [review]
👍 nice work 💯

**Pull request merged**